adapters/Maio/MaioAdapter/RTBAdapter/GADRTBAdapterMaioEntryPoint.m

```diff
@@ -0,0 +1,32 @@
+//
+//  GADRTBAdapterMaio.m
+//  Adapter
+//
+//  Created by i-mobile on 2020/11/18.
+//  Copyright © 2020 Google. All rights reserved.
+//
+
+#import "GADRTBAdapterMaioEntryPoint.h"
+#import "GADRTBMaioRewardedAd.h"
+#import "GADRTBMaioInterstitialAd.h"
+
+@implementation GADRTBAdapterMaioEntryPoint {
+  GADRTBMaioRewardedAd *_rewarded;
+  GADRTBMaioInterstitialAd *_interstitial;
+}
+
+- (void)collectSignalsForRequestParameters:(nonnull GADRTBRequestParameters *)params completionHandler:(nonnull GADRTBSignalCompletionHandler)completionHandler {
+  completionHandler(nil, nil);
+}
+
+-(void)loadRewardedAdForAdConfiguration:(nonnull GADMediationRewardedAdConfiguration *)adConfiguration completionHandler:(nonnull GADMediationRewardedLoadCompletionHandler)completionHandler {
+  _rewarded = [[GADRTBMaioRewardedAd alloc] init];
+  [_rewarded loadRewardedAdForAdConfiguration:adConfiguration completionHandler:completionHandler];
```

Comment:
Can we update the init and `load` methods such as the init method will take in the `adConfiguration` and the load method will take the `completionHandler`, as shown below?

```
  _rewarded = [[GADRTBMaioRewardedAd alloc] initWithAdConfiguration:adConfiguration];
  [_rewarded loadRewardedAdWithCompletionHandler:completionHandler];
```

Please apply the same below for interstitial ads.

Reply:
Certainly.